src/tenant/tenant.resolver.ts

```diff
@@ -45,4 +45,35 @@ export default class TenantResolver {
 
     return tenant;
   }
+
+  @Mutation(() => Boolean)
+  async markRentAsLate(@Arg('id', () => Int) id: number): Promise<boolean> {
+    const t = await Tenant.findOne({where: {id}});
```

Comment:
As for other mutation, use defined error check and fail if the user does not exists: 
```suggestion
        const tenant = await Tenant.findOne({ where: { id: data.id } });
    if (!tenant) {
      throw new Error(TENANT_ERRORS.INVALID_ID);
    }
```

Also this prevent having to nest the rest of your function in `if(t)`

Reply:
Since we start to repeat this code quite a lot, maybe extract it to a `findTenantOrThrow` function